common/src/main/resources/assets/voicechat/lang/es_es.json

```diff
@@ -1,213 +1,217 @@
+  "cloth_config.voicechat.config.voice_deactivation_delay": "Retraso de Desactivación",
+  "cloth_config.voicechat.config.voice_deactivation_delay.description": "El tiempo que tarda el micrófono en desactivarse\ncuando se utiliza el método de activación por voz.\n\n§81 ► 20 milisegundos\n§82 ► 40 milisegundos\n§83 ► 60 milisegundos\n§8...",
+  "cloth_config.voicechat.config.output_buffer_size": "Tamaño del búfer de Salida",
+  "cloth_config.voicechat.config.output_buffer_size.description": "El tamaño del búfer de salida de sonido en paquetes.\n\n§7Valores más altos significan una mayor latencia, pero menos chasquidos.\n§7Aumenta este valor si tienes una conexión a Internet inestable.",
+  "cloth_config.voicechat.config.hud_icon_scale": "Escala de los Iconos",
+  "cloth_config.voicechat.config.hud_icon_scale.description": "La escala de los iconos en pantalla.",
+  "cloth_config.voicechat.config.hud_icon_pos_x": "Posición X del Icono",
+  "cloth_config.voicechat.config.hud_icon_pos_x.description": "La posición X de los iconos en pantalla.\nLos valores negativos significan anclaje a la derecha.",
+  "cloth_config.voicechat.config.hud_icon_pos_y": "Posición Y del Icono",
+  "cloth_config.voicechat.config.hud_icon_pos_y.description": "La posición Y de los iconos en pantalla.\nLos valores negativos significan anclaje abajo.",
+  "cloth_config.voicechat.config.group_player_icon_orientation": "Orientación de los Iconos de Grupo",
+  "cloth_config.voicechat.config.group_player_icon_orientation.description": "La orientación de los iconos de los jugadores del grupo en pantalla.",
+  "cloth_config.voicechat.config.group_player_icon_orientation.vertical": "Vertical",
+  "cloth_config.voicechat.config.group_player_icon_orientation.horizontal": "Horizontal",
+  "cloth_config.voicechat.config.group_hud_icon_scale": "Escala de los Iconos del Grupo",
+  "cloth_config.voicechat.config.group_hud_icon_scale.description": "La escala de los iconos del grupo.",
+  "cloth_config.voicechat.config.group_player_icon_pos_x": "Posición X del Icono Grupo",
+  "cloth_config.voicechat.config.group_player_icon_pos_x.description": "Posición X de los iconos de los jugadores del grupo.\nLos valores negativos significan anclaje a la derecha.",
+  "cloth_config.voicechat.config.group_player_icon_pos_y": "Posición Y del Icono Grupo",
+  "cloth_config.voicechat.config.group_player_icon_pos_y.description": "Posición Y de los iconos de jugador del grupo.\nLos valores negativos significan anclaje abajo.",
+  "cloth_config.voicechat.config.show_own_group_icon": "Mostrar el Icono del propio Grupo",
+  "cloth_config.voicechat.config.show_own_group_icon.description": "Si se debe mostrar el icono del grupo, cuando te encuentras en un grupo."
```

Comment:
I won't accept translations that add colors or arrows. This should never be part of a translation.